src/pages/stacks-blockchain/overview.md

```diff
@@ -89,7 +51,24 @@ An example of this is Stacking.
 
 ### Stacking
 
-Stackers are Stacks (STX) Token holders who provide a valuable service to the network by locking up their STX for a certain
+Stackers are Stacks (Stacks) Token holders who provide a valuable service to the network by locking up their Stacks for a certain
 period of time. As a reward, Stackers receive the Bitcoin that miners transfer as part of Proof of Transfer (PoX).
 
 -> Proof of Transfer and Stacking are in active development and are coming soon
+
+## Design specification
+
+Stacks improvement proposals (SIPs) are aimed at describing the implementation of the Stacks blockchain, as well as
+proposing improvements. They should contain concise technical specifications of features or standards and the rationale
+behind it. SIPs are intended to be the primary medium for proposing new features, for collecting community input on a
+system-wide issue, and for documenting design decisions.
+
+-> See [SIP 000](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-000-stacks-improvement-proposal-process.md) for more details about the SIPs process.
+
+- ✅ [SIP 001:](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-001-burn-election.md) Burn Election
+- ✅ [SIP 002:](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-002-smart-contract-language.md) Clarity, a language for predictable smart contracts
+- ✅ [SIP 003:](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-003-peer-network.md) Peer Network
+- ✅ [SIP 004:](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-004-materialized-view.md) Cryptographic Committment to Materialized Views
+- ✅ [SIP 005:](https://github.com/blockstack/stacks-blockchain/blob/master/sip/sip-005-blocks-and-transactions.md) Blocks, Transactions, and Accounts
+- [ ] SIP 006: Clarity Execution Cost Assessment _(Q2 2020)_
```

Comment:
[nit] you can actually link to SIP 006, 007 and 008 -- they're all written.

Reply:
I actually made this a table with *proposed* and *active* indicators. 

- SIP 6 is not proposed
- SIP 7 is proposed but not active
- SIP 8 - I couldn't find in [the sip folder](https://github.com/blockstack/stacks-blockchain/tree/develop/sip). any pointers?

Reply:
thanks!